app/views/home/how_government_works.html.erb

```diff
@@ -18,7 +18,7 @@
       <div class="halves-width-section prime-minister">
         <div class="one-half">
           <div class="inner-block">
-            <% if @prime_minister.present? %>
+            <% if @prime_minister.present? && @prime_minister.image_url(:s465).present? %>
```

Comment:
Presumably the real data has an image for the prime minister? Was this change because of a test failure?